src/domain/alerts/mappers/multi-send.mapper.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@nestjs/common';
+import {
+  decodeFunctionData,
+  getAddress,
+  Hex,
+  hexToBigInt,
+  hexToNumber,
+  parseAbi,
+  size,
+  slice,
+} from 'viem';
+
+@Injectable()
+export class MultiSendMapper {
+  private readonly abi = parseAbi([
+    'function multiSend(bytes memory transactions)',
+  ]);
+
+  mapMultiSendTransactions(multiSendData: Hex): Array<{
+    operation: number;
+    to: string;
+    value: bigint;
+    data: Hex;
+  }> {
+    const mapped: Array<{
+      operation: number;
+      to: string;
+      value: bigint;
+      data: Hex;
+    }> = [];
+
+    const multiSend = decodeFunctionData({
+      abi: this.abi,
+      data: multiSendData,
+    });
+    const transactions = multiSend.args[0];
+
+    let i = 0;
```

Comment:
What do you think about having a more explicit name for this variable, like `cursor`, `pointer`, etc.?

Reply:
I've changed it to `cursor` in f78093192d8c7d5555f13003c7548c368e3a20b0.